src/components/ScholarshipAmountField.jsx

```diff
@@ -0,0 +1,117 @@
+import React from 'react';
+import {
+  FormLabel,
+  Radio,
+  RadioGroup,
+  FormControl,
+  FormControlLabel,
+} from '@material-ui/core';
+import { makeStyles } from '@material-ui/core/styles';
+import PropTypes from 'prop-types';
+import AmountType from '../types/AmountType';
+import AmountTextField from './AmountTextField';
+
+const useStyles = makeStyles((theme) => ({
+  spanStyle: {
+    margin: theme.spacing(1),
+  },
+  disabledSpan: {
+    opacity: 0.5,
+  },
+}));
+
+function ScholarshipAmountField(props) {
+  const classes = useStyles();
+  const {
+    amountType,
+    minAmount,
+    maxAmount,
+    onTypeChange,
+    updateAmount,
+  } = props;
+
+  function displayAmountFields(option) {
+    switch (option) {
+      case AmountType.Range:
+        return (
+          <>
+            <AmountTextField
+              label="Minimum Amount"
+              value={minAmount || ''}
```

Comment:
This is to that the `placeholder` shows, without it the default value `0` would show. 
What I could also do is let the value be a string and then when verifying the input I can convert it to a number.